frontend/src/scenes/settings/project/ProjectSettings.tsx

```diff
@@ -141,6 +142,21 @@ export function ProjectVariables(): JSX.Element {
                 </p>
                 <CodeSnippet thing="project ID">{String(currentTeam?.id || '')}</CodeSnippet>
             </div>
+            {['https://eu.posthog.com', 'https://us.posthog.com'].includes(
+                preflightLogic.values.preflight?.site_url ?? ''
+            ) ? (
+                <div className="flex-1">
+                    <h3 id="project-id" className="min-w-[25rem]">
+                        Project Region
+                    </h3>
+                    <p>This is the region where your PostHog data is currently hosted in.</p>
```

Comment:
```suggestion
                    <p>This is the region where your PostHog data is hosted.</p>
```

Currently makes it seem changeable, which it isn't.